docs/docs/recipes/gitlab-continuous-integration.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Recipes: Continuous Integration on Gitlab"
+tableOfContentsDepth: 1
+---
+
+Push your code to gitlab and automate your production build!
+
+### Prerequisites
+
+- Make sure you have the [Gatsby CLI](https://www.gatsbyjs.org/docs/gatsby-cli) installed
+- A [Gitlab](https://gitlab.com/) account 
+
+### Directions
+1. Create a gatsby site
+```shell
+gatsby new {your-project-name}
+```
+2. Change directory and start a development server
+```shell
+cd {your-project-name}
+yarn develop
+```
+
+3. Stop your development server (`Ctrl + C` on your command line in most cases)
+
+4. Create a `.gitlab-ci.yml` with the following content:
+
+```
+image: node:12.16.1
+
+cache:
+  paths:
+    - node_modules/
+
+stages:
+  - build
+
+build:
+  stage: build
+  script:
+    - yarn
+    - yarn build
```

Comment:
Using this configuration returns a YAML invalid error, I added a few parameters to make sure the demo will build. 
```suggestion
    image: node:latest

    stages:
      - build

    cache:
      paths:
        - node_modules/

    install_dependencies:
      stage: build
      script:
        - npm install
      artifacts:
        paths:
          - node_modules/
```